XenModel/Messages.resx

```diff
@@ -10657,6 +10660,12 @@ Please reconnect the host and try again</value>
   <data name="PROBING_HBA_TITLE" xml:space="preserve">
     <value>Probing for LUNs on {0}</value>
   </data>
+  <data name="PROBLEM_HEALTH_CHECK_SERVICE_DESCRIPTION" xml:space="preserve">
+    <value>{0}: The Health Check service has been removed in [XenServer] {1}.</value>

```

Comment:
Could you please also change `[XenServer] {1}` to `{0} {1}` here and below, and format it with the BrandManager variables? We should start migrating from rebranding with sed.

Reply:
Fixed in second commit.